Yoast/wordpress-seo
# Add unit tests to increase indexables coverage

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Increases coverage for the indexables code.

## Relevant technical choices:

* I also added a capabilities check (`current_user_can`) for the `Indexation_Permalink_Warning_Presenter`, to make it equal to all the other presenters.
* In `tests/integrations/watchers/indexable-category-permalink-watcher-test.php`, I didn't manage to get `$wpdb` working from within the `setUp` function. If it's an easy fix, this would be nice to add. Otherwise it's probably not worth it.

## Test instructions

This PR can be tested by following these steps:

* See all the unit tests pass in Travis.
* See 100% code coverage for `indexable-term-indexation-action.php`, `src/integrations/admin/indexation-integration.php`, `src/integrations/watchers/indexable-category-permalink-watcher.php` and `src/presenters/admin/indexation-permalink-warning-presenter.php`. 

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/P2-27
